airgun/views/partitiontable.py

```diff
@@ -40,51 +42,48 @@ def is_displayed(self):
 
 class PartitionTableEditView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
-    name = TextInput(locator="//input[@id='ptable_name']")
-    default = Checkbox(id='ptable_default')
-    template = ACEEditor()
-    audit_comment = TextInput(id="ptable_audit_comment")
     submit = Text('//input[@name="commit"]')
 
-    snippet = Checkbox(locator="//input[@id='ptable_snippet']")
-    os_family_selection = ConditionalSwitchableView(reference='snippet')
+    @View.nested
+    class template(SatTab):
+        name = TextInput(id='ptable_name')
+        default = Checkbox(id='ptable_default')
+        snippet = Checkbox(locator="//input[@id='ptable_snippet']")
+        os_family_selection = ConditionalSwitchableView(reference='snippet')
+
+        @os_family_selection.register(True)
+        class SnippetOption(View):
+            pass
```

Comment:
Do we still need to register this even if there is no content is this caused by some ConditionalSwitchableView execption?

Reply:
hmm, it was there initially. Probably, I can test without

Reply:
yeah. That will produce:
```
ValueError: Could not find a corresponding registered view.
```